Tests/test_Seq_objs.py

```diff
@@ -737,6 +737,165 @@ def test_init_typeerror(self):
         self.assertRaises(TypeError, Seq, (1066))
         self.assertRaises(TypeError, Seq, (Seq("ACGT", generic_dna)))
 
+    def test_join_Seq(self):
+        """Checks if Seq join correctly concatinates sequence with the spacer."""
+        # Only expect it to take Seq objects and/or strings in an iterable!
+
+        filename = 'Fasta/f003'
+        seqlist = SeqIO.parse(filename, 'fasta')
```

Comment:
Can you move the file based example into a separate set method, e.g. ``def test_join_with_file(self):``

Reply:
Also, ``seqlist`` is not a list of ``Seq`` objects. It is a generator yielding ``SeqRecord`` objects. How about:

```python
seq_list = [record.seq for record in SeqIO.parse(filename, 'fasta')]
```